.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,48 @@
+name: Windows CI
+on: [pull_request]
+
+jobs:
+  run-windows-tests:
+    name: Build & run tests
+    runs-on: windows-2022
+
+    steps:
+    - uses: actions/checkout@v3
+      name: Checkout Code
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '^14'
+        cache: 'yarn'
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.1.3
+      with:
+        vs-version: '[17.0,)'
+        msbuild-architecture: x64
+
+    - name: Install node modules
+      run: yarn --pure-lockfile

```

Comment:
I believe the command should be `yarn install --pure-lockfile`, though i am not sure why you are trying to stop it from generating a lockfile, considering it is going to be thrown out as soon as the CI job is done

Reply:
We use `--frozen-lockfile` to pin the dependencies so that if some minor version is ever bumped, we don't start seeing unrelated issues